numpy/testing/nosetester.py

```diff
@@ -326,9 +325,9 @@ def test(self, label='fast', verbose=1, extra_argv=None,
              <http://nedbatchelder.com/code/modules/coverage.html>`_).
         raise_warnings : str or sequence of warnings, optional
             This specifies which warnings to configure as 'raise' instead
-            of 'warn' during the test execution.  Valid strings are:
+            of 'always' during the test execution.  Valid strings are:
 
-              - "develop" : equals ``(DeprecationWarning, RuntimeWarning)``
+              - "develop" : equals ``(Warning,)``
```

Comment:
So this is why you removed the check for any warnings above? It seems a bit fragile to depend on the setting here. If someone were to change this they would never notice that some tests lost a fraction of their validity.